apps/e2e-test/src/playwright-support/app-specific/element-verification.ts

```diff
@@ -53,3 +53,16 @@ export const verifyOptions = async (
     expect(actOptions).toEqual(options);
   }
 };
+
+export const verifyOptionsContains = async (
+  page: Page,
+  selector: TargetSelector,
+  options: string[],
+  negate = false
```

Comment:
Please remove `negate` from arguments, we'd better use another function called `verifyOptionsNotContains` which is more explicit.

Reply:
@vincentsong Fixed.